tklinenums/tklinenums.py

```diff
@@ -37,6 +37,7 @@ def __init__(
         master: Misc,
         textwidget: Text,
         justify: str = "left",
+        ttk_theme: bool = False,
```

Comment:
```suggestion
        # None means take colors from text widget (default).
        # Otherwise it is a function that takes no arguments and returns (fg, bg) tuple.
        color_provider: Callable[[], tuple[str, str]] | None = None,
```